bin/deploy.sh

```diff
@@ -0,0 +1,28 @@
+#! /bin/bash
+
+# Tag, Push and Deploy only if it's not a pull request
+if [ -z "$TRAVIS_PULL_REQUEST" ] || [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+
+    # Push only if we're testing the staging branch
+    #if [ "$TRAVIS_BRANCH" == "staging" ]; then
```

Comment:
Is "staging" the global recommendation for the branch we want development to merge to, and to deploy from?  Why are we abandoning the well-known "master"?  Most teams seemed to converge on a "merge-to-master" pattern last year fairly well.

Reply:
I don't know. Its a relic from copying it form the old pattern from 2017. I am not aware of the officially recommended practice, but will follow whatever you guys set the standard as.

Reply:
Staging is just intended to be place to get a deploy ready code together, confirm no merge conflicts and the like. Intention is still that merge-to-master should trigger deploy

Reply:
when we setup the transportation repo, Merritt and I discussed just setting staging as the default branch, basically with the 2 PRs could merge. 

Intention was that merge-to-master will be what sets deploy off, with only admins able to approve the PR.